docs(influxdb3): document duplicate point write ordering and workarounds (#6584)
influxdata/docs-v2: content/shared/v3-line-protocol.md

ORDER BY time DESC
```

### Anti-patterns to avoid

The following patterns will produce non-deterministic results when duplicate points are flushed together:

#### ❌ Overwriting the same (time, tags) point

**Don't do this**:

```text
-- All writes use the same timestamp
device_status,device_id=sensor01 status="active",temperature=72.5 1700000000000000000
device_status,device_id=sensor01 status="active",temperature=73.1 1700000000000000000
device_status,device_id=sensor01 status="inactive",temperature=73.1 1700000000000000000
```

**Problem**: When these writes are flushed together, any of the three values might be retained.

#### ❌ Adding a field while overwriting the same (time, tags)

**Don't do this**:

```text
-- All writes use the same timestamp, but add a version field
device_status,device_id=sensor01 status="active",temperature=72.5,version=1i 1700000000000000000
device_status,device_id=sensor01 status="active",temperature=73.1,version=2i 1700000000000000000
device_status,device_id=sensor01 status="inactive",temperature=73.1,version=3i 1700000000000000000
```

**Problem**: The points are still duplicates (same time and tags).

> Adding a field doesn't make them unique points.

Reviewer: ````suggestion
Adding a field doesn't make a points unique.
Points with the same time and tag set are still considered duplicates--for example,
**don't do this**:

```text
-- All writes use the same timestamp, but add a version field
device_status,device_id=sensor01 status="active",temperature=72.5,version=1i 1700000000000000000
device_status,device_id=sensor01 status="active",temperature=73.1,version=2i 1700000000000000000
device_status,device_id=sensor01 status="inactive",temperature=73.1,version=3i 1700000000000000000
```
````

> #### ❌ Relying on write delays to force ordering

Reviewer: ```suggestion
#### Don't rely on write delays to force ordering
```

**Don't do this**:

```text
-- Writing with delays between each write
device_status,device_id=sensor01 status="active" 1700000000000000000
# Wait 10 seconds...
device_status,device_id=sensor01 status="inactive" 1700000000000000000
```

**Problem**: Delays don't guarantee that duplicate points won't be flushed together.

> The flush interval depends on buffer size, ingestion rate, and system load.

Reviewer: ````suggestion
Delays don't guarantee that duplicate points won't be flushed together.
The flush interval depends on buffer size, ingestion rate, and system load.

For example, **don't do this**:

```text
-- Writing with delays between each write
device_status,device_id=sensor01 status="active" 1700000000000000000
# Wait 10 seconds...
device_status,device_id=sensor01 status="inactive" 1700000000000000000
```
````
